plugin/al/lib/AL_USDMaya/AL/usdmaya/fileio/translators/TranslatorContext.h

```diff
@@ -32,7 +32,7 @@
 
 #include <vector>
 #include <string>
-#include "AL/usd/utils/ForwardDeclares.h"
+#include <usdUtils/ForwardDeclares.h>
```

Comment:
This seems inconsistent with the other include convention which would look like
```
#include "ForwardDeclares.h"
```
again - I would argue for keeping the existing convention pending discussion

Reply:
Not sure I understand, ForwardDeclares.h is now moved to UsdUtils